app/views/embed/loading.html.erb

```diff
@@ -1,12 +1,9 @@
+<% content_for :head do %>
+  <meta http-equiv="refresh" content="30">
```

Comment:
The unit here is seconds, while `setTimeout` takes milliseconds. Thus 30_000 => 30.